File: tests/fixtures/usage-graph-csharp/Outer.cs
namespace Example
{
    public class Outer
    {
        public class Inner
        {
            public int Compute()
            {
                // Unqualified call must attribute to the enclosing nested class.
                return Helper();
            }

            public int Helper()
            {
                return 1;
            }
        }
    }
}

File: tests/fixtures/usage-graph-java/com/example/Outer.java
package com.example;

public class Outer {
    public static class Inner {
        public int compute() {
            // Unqualified call: attributes to the enclosing nested class
            // `com.example.Outer.Inner`, not a same-named top-level type.
            return helper();
        }

        public int helper() {
            return 1;
        }
    }
}
